src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -938,6 +943,17 @@ def get_environment_client(self):
         else:
             return ManagedEnvironmentPreviewClient
 
+    def set_up_runtime(self):
```

Comment:
Here should check if arguments exist:
if self.get_argument_runtime() is not None or self.get_argument_enable_java_metrics() is not None: